components/suggest/src/db.rs

```diff
@@ -365,4 +369,42 @@ impl<'a> SuggestDao<'a> {
         )?;
         Ok(())
     }
+
+    /// update last ingest if record last modified is newer than the existing one recorded
```

Comment:
nitpick: Let's follow the [Rustdoc conventions](https://github.com/rust-lang/rfcs/blob/master/text/1574-more-api-documentation-conventions.md#appendix-a-full-conventions-text) for docstrings, and make this a complete sentence.

```suggestion
    /// Update the last ingest timestamp if the given last modified time is newer than the existing one recorded.
```